Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM docker:1.10-dind
+FROM docker:17.06.1-ce-rc1-dind
 
-RUN apk update && apk add --no-cache bash curl jq parted btrfs-progs docker util-linux
+RUN apk update && apk add --no-cache python3 parted btrfs-progs docker util-linux inotify-tools
+RUN pip3 install sh==1.12.14 requests==2.18.1
```

Comment:
Can the python dependencies be put in a `requirements.txt` and installed from there? It would be more maintainable/transparent, than having it inline installed in the `Dockerfile`, imho.